inject/inject.vala

```diff
@@ -225,6 +225,41 @@ namespace Frida.Inject {
 
 			exit_code = 0;
 
+			var fd = stdin.fileno();
+#if WINDOWS
+			var inchan = new IOChannel.win32_new_fd (fd);
+#else
+			var inchan = new IOChannel.unix_new (fd);
+#endif
+			inchan.add_watch (IOCondition.IN, (source, condition) => {
+				string str_return = null;
```

Comment:
I'd move this to right before `try {` so it makes it clear what's about to happen inside the `try`.

Reply:
I don't think I can move this? `inchan` is used to call `add_watch` so it needs to exist outside the callback. The try needs to handle potential errors occurring inside the callback.

Reply:
I meant the `str_return` variable, sorry for being unclear :)

Reply:
Ok thanks.